src/store/recordAction.ts

```diff
@@ -0,0 +1,11 @@
+import * as FullStory from '@fullstory/browser';
+import { Middleware } from 'redux';
+
+const recordAction: Middleware = () => next => action => {
+  if (typeof action !== 'function') {
```

Comment:
Might be safer to change this to guard against what we're logging:

```suggestion
  if (action && typeof action.type === 'string') {
```